### ISSUE TITLE
Completion bug in Export operation

### ISSUE THREAD

**Issue by Krasnyanskiy** [Jaspersoft/jrsh#164]
I've pressed a `Tab` key after `~/Dev`
```bash
$> export repository /
organizations/   public/          temp/            themes/          
$> export repository ~/Dev
```
and app gave me an error
```bash
Exception in thread "main" com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.IllegalParameterValueException: The value "~/Dev" for parameter "folderUri" is invalid.
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
        at com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.handling.DefaultErrorHandler.handleBodyError(DefaultErrorHandler.java:95)
        at com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.handling.DefaultErrorHandler.handleError(DefaultErrorHandler.java:67)
        at com.jaspersoft.jasperserver.jaxrs.client.core.JerseyRequest.executeRequest(JerseyRequest.java:142)
        at com.jaspersoft.jasperserver.jaxrs.client.core.JerseyRequest.executeRequest(JerseyRequest.java:130)
        at com.jaspersoft.jasperserver.jaxrs.client.core.JerseyRequest.get(JerseyRequest.java:99)
        at com.jaspersoft.jasperserver.jaxrs.client.apiadapters.resources.BatchResourcesAdapter.search(BatchResourcesAdapter.java:48)
        at com.jaspersoft.jasperserver.jrsh.core.completion.impl.JrshRepositoryCompleter$Downloader.download(JrshRepositoryCompleter.java:241)
        at com.jaspersoft.jasperserver.jrsh.core.completion.impl.JrshRepositoryCompleter.complete(JrshRepositoryCompleter.java:47)
        at jline.console.completer.ArgumentCompleter.complete(ArgumentCompleter.java:152)
        at jline.console.completer.AggregateCompleter$Completion.complete(AggregateCompleter.java:121)
        at jline.console.completer.AggregateCompleter.complete(AggregateCompleter.java:80)
        at jline.console.ConsoleReader.complete(ConsoleReader.java:3261)
        at jline.console.ConsoleReader.readLine(ConsoleReader.java:2621)
        at jline.console.ConsoleReader.readLine(ConsoleReader.java:2269)
        at jline.console.ConsoleReader.readLine(ConsoleReader.java:2257)
        at com.jaspersoft.jasperserver.jrsh.core.evaluation.strategy.impl.ShellEvaluationStrategy.eval(ShellEvaluationStrategy.java:50)
        at com.jaspersoft.jasperserver.jrsh.runner.App.main(App.java:14)
```

Despite the fact that `~/Dev` is a wrong repository path, I don't want to see that stacktrace in any case! So we need to hide it. If I try to complete the wrong repository path then App should ignore all errors that occur.

**Comment by Krasnyanskiy** [comment]
These changes should resolve the issue:
```java
public boolean isResourceExist(String path) {
    try {
        SessionFactory.getSharedSession()
                .resourcesService()
                .resources()
                .parameter(FOLDER_URI, path)
                .parameter(RECURSIVE, "false")
                .search();
    } catch (ResourceNotFoundException e2) {
        log.debug("resource not found");
        return false;
    } catch (NullPointerException e) {
        log.debug("NPE, returning empty list");
        return false;
    } catch (IllegalParameterValueException e) {
        // bug #164 fix
        return false;
    }
    return true;
}
```

**Comment by Krasnyanskiy** [comment]
Related to the [pull](https://github.com/Jaspersoft/jrsh/pull/174).

**Comment by Krasnyanskiy** [comment]
Fixed in (v2.0.6)[https://github.com/Jaspersoft/jrsh/releases/tag/v2.0.6]
